core/templates/dev/head/player/PlayerServices.js

```diff
@@ -279,6 +279,12 @@ oppia.factory('oppiaPlayerService', [
     getExplorationTitle: function() {
       return exploration.title;
     },
+    getExplorationVersion: function() {
+      return version;
+    },
+    getStateContent: function(stateName) {
+      return exploration.getState(stateName).content;
```

Comment:
Can this be refactored slightly to:

    getStateContentHtml: function(stateName) {
      return exploration.getContentHtml(stateName);
    }

where the result is a string? I think all the [0] and 'value' and '.content' stuff should be encapsulated within the Exploration object.